device-types/Cisco/N3K-C3172PQ-10GE.yaml

```diff
@@ -1,14 +1,18 @@
 ---
 manufacturer: Cisco
-model: Nexus 3172PQ-10GE
-slug: cisco-n3k-c3172q-10ge
-part_number: N3K-C3172Q-10GE
+model: Nexus C3172PQ-10GE
+part_number: N3K-C3172PQ-10GE
+slug: cisco-n3k-c3172pq-10ge
 u_height: 1
 is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb0
+    type: usb-mini-b
```

Comment:
Why was this re-ordered?

Reply:
Moved beneath interfaces block, before module-bays block